src/pages/recipes/arguments/index.md

```diff
@@ -0,0 +1,70 @@
+---
+keywords:
+  - UXP Arguments
+  - InDesign Server Arguments
+  - DoScript Arguments
+---
+# Passing Arguments
```

Comment:
Would I be able to pass arguments via InDesign app UI?

Reply:
No, as discussed in the meeting adding an info block that clarifies this would be sufficient.